ansible_base/lib/serializers/common.py

```diff
@@ -71,8 +71,18 @@ def update(self, instance, validated_data):
         return super().update(instance, validated_data)
 
 
+class CommonModelSerializer(AbstractCommonModelSerializer):
+    class Meta(AbstractCommonModelSerializer.Meta):
+        fields = ['id', 'url', 'created', 'created_by', 'modified', 'modified_by', 'related', 'summary_fields']
+
+
 class NamedCommonModelSerializer(CommonModelSerializer):
     class Meta(CommonModelSerializer.Meta):
         fields = [
             'name',
         ] + CommonModelSerializer.Meta.fields
+
+
+class ImmutableCommonModelSerializer(AbstractCommonModelSerializer):
+    class Meta(AbstractCommonModelSerializer.Meta):
+        fields = ['id', 'url', 'created', 'created_by', 'related', 'summary_fields']
```

Comment:
I suggest putting `['id', 'url', 'created', 'created_by', 'related', 'summary_fields']` in a tuple defined outside of these classes, and then have `CommonModelSerializer` add to it to avoid duplication.